packages/aft/lib/src/commands/docs_command.dart

```diff
@@ -0,0 +1,363 @@
+      createStaticHandler(outputDir, defaultDocument: 'index.html'),
+      InternetAddress.anyIPv4,
+      port,
+    );
+    logger
+      ..info('Serving docs on http://localhost:${server.port}')
+      ..info('Press Ctrl+C to stop.');
+
+    // Wait for interruption
+    await StreamGroup.merge([
+      ProcessSignal.sigint.watch(),
+      ProcessSignal.sigterm.watch(),
+    ]).first;
+
+    // Shutdown
+    logger.debug('Shutting down...');
+    await Future.wait(watchers.map((watcher) => watcher.close()));
+    await server.close(force: true);
+  }
+}
+
+/// Watches a [package] for changes and calls [onChange] whenever a file is changed.
+class _PackageWatcher with Closeable {
+  _PackageWatcher(this.package, {required this.onChange}) {
+    _listenForEvents();
+  }
+
+  final PackageInfo package;
+  final Future<void> Function() onChange;
+  late final StreamSubscription<FileSystemEvent> _subscription;
+
+  void _listenForEvents() {
+    final directoriesToWatch = [
+      for (final subpath in const ['doc/lib', 'lib'])
+        Directory(p.join(package.path, subpath)),
+    ].where((dir) => dir.existsSync());
+    final eventStream = StreamGroup.merge([
+      for (final directory in directoriesToWatch)
+        directory.watch(
+          events: FileSystemEvent.all,
+          recursive: true,
+        ),
+    ]);
+    Future<void>? currentOperation;
+    _subscription = eventStream.listen((_) {
+      currentOperation ??= onChange().whenComplete(() {
+        currentOperation = null;
+      });
+    });
+  }
+
+  @override
+  Future<void> close() => _subscription.cancel();
+}
+
+/// Index page template to make navigation between package docs easier.
+const _indexTmpl = r'''
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Amplify Flutter</title>
+    <style>
+        body {
+            font-family: Roboto, sans-serif;
+            text-align: center;
+        }
+
+        #docs {
+            width: 100%;
+            max-width: 800px;
+            margin: 0 auto;
+            display: flex;
+            flex-direction: column;
+            align-items: stretch;
+            overflow-x: auto;
+        }
+
+        table {
+            table-layout: auto;
+            word-wrap: break-word;
+            border-collapse: collapse;
+        }
+
+        th,
+        td {
+            padding: 8px;
+            text-align: left;
+            border-bottom: 1px solid #ddd;
+        }
+
+        th {
+            background-color: #f2f2f2;
+            font-weight: bold;
+        }
+
+        tr:nth-child(even) {
+            background-color: #f9f9f9;
+        }
+
+        tr:hover {
+            background-color: #eaeaea;
+        }
+
+    </style>
+</head>
+
+<body>
+    <section id="docs">
+        <h1>Amplify Docs</h1>
+        <table>
```

Comment:
Made the first column sticky while scrolling horizontally :P 

```suggestion
            overflow-x: auto;
        }

        table {
            table-layout: auto;
            word-wrap: break-word;
            border-collapse: collapse;
        }

        th,
        td {
            padding: 8px;
            text-align: left;
            border-bottom: 1px solid #ddd;
        }

        thead th:first-child {
            background-color: #f2f2f2;
            position: sticky;
            left: 0;
        }

        td:first-child {
            position: sticky;
            left: 0;
        }

        th {
            background-color: #f2f2f2;
            font-weight: bold;
        }

        tr:nth-child(even) {
            background-color: #f9f9f9;
        }

        tr:nth-child(even) > td:first-child {
            background-color: #f9f9f9;
        }

        tr:nth-child(odd) {
            background-color: white;
        }

        tr:nth-child(odd) > td:first-child {
            background-color: white;
        }

        tr:hover {
            background-color: #eaeaea
        }

        tr:nth-child(even):hover > td:first-child, tr:nth-child(odd):hover > td:first-child {
            background-color: #eaeaea;
        }
    </style>
</head>

<body>
    <h1>Amplify Docs</h1>
    <section id="docs">
        <table>
```

Reply:
Lol good stuff - thanks!!